Makefile

```
@@ -59,6 +59,9 @@ testnsxv:
 testtm:
 	cd govcd && go test -tags "tm" -timeout 0  -check.vv
 
+testcci:
+	cd govcd && GOVCD_API_VERSION=40.0 go test -vcd-test-org-user -tags cci -check.vv -timeout=0
```

Not needed once https://github.com/vmware/go-vcloud-director/pull/748 is merged
```suggestion
	cd govcd && go test -vcd-test-org-user -tags cci -check.vv -timeout=0
```